std/file.d

```diff
@@ -4971,6 +4974,26 @@ auto dFiles = dirEntries("","*.{d,di}",SpanMode.depth);
 foreach (d; dFiles)
     writeln(d.name);
 --------------------
+To handle subdirectories with denied read permission, use `SpanMode.shallow`:
```

Comment:
Maybe this is the problem? Maybe you could put this sentence as a comment in the code?